services/ledger_service.go

```diff
@@ -8,9 +8,8 @@ import (
 
 	"google.golang.org/grpc/credentials"
 
-	didTypes "github.com/cheqd/cheqd-node/x/did/types"
-	didUtils "github.com/cheqd/cheqd-node/x/did/utils"
-	resource "github.com/cheqd/cheqd-node/x/resource/types"
+	didTypes "github.com/cheqd/cheqd-node/api/v2/cheqd/did/v2"
+	resource "github.com/cheqd/cheqd-node/api/v2/cheqd/resource/v2"
```

Comment:
let's rename this alias to `resourceTypes` for consistency.

Reply:
Done! ✅